packages/cli/src/tools/config/readConfigFromDisk.js

```diff
@@ -118,7 +155,6 @@ function readLegacyDependencyConfigFromDisk(
       : {},
   };
 
-  // @todo: paste a link to documentation that explains the migration steps
```

Comment:
removed the todo, since migration guide is displayed when reading project config (so it displays for projects and library maintainers, but not to user's deps which would be annoying in the transition period)